modules/exploits/windows/local/cve_2022_26904_superprofile.rb

```diff
@@ -0,0 +1,254 @@
+      return CheckCode::Appears('Vulnerable Windows 10 21H1 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.19042.0')) # Windows 10 20H2 / Windows Server, Version 20H2
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 20H2 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.19041.0')) # Windows 10 v2004 / Windows Server v2004
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 v2004 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.18363.0')) # Windows 10 v1909 / Windows Server v1909
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 v1909 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.18362.0')) # Windows 10 v1903
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 v1903 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.17763.0')) # Windows 10 v1809 / Windows Server 2019 v1809
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 v1809 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.17134.0')) # Windows 10 v1803
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 v1803 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.16299.0')) # Windows 10 v1709
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 v1709 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.15063.0')) # Windows 10 v1703
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 v1703 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.14393.0')) # Windows 10 v1607 / Windows Server 2016 v1607
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 v1607 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.10586.0')) # Windows 10 v1511
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 v1511 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('10.0.10240.0')) # Windows 10 v1507
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 10 v1507 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('6.3.9600.0')) # Windows 8.1/Windows Server 2012 R2
+      target_not_presently_supported
+      return CheckCode::Appears('Vulnerable Windows 8.1/Windows Server 2012 R2 build detected!')
+    elsif (build_num_gemversion >= Rex::Version.new('6.2.9200.0')) # Windows 8/Windows Server 2012
+      target_not_presently_supported
+      return CheckCode::AppearsAppears('Vulnerable Windows 8/Windows Server 2012 build detected!')
```

Comment:
```suggestion
      return CheckCode::Appears('Vulnerable Windows 8/Windows Server 2012 build detected!')
```

Reply:
Woops, typos! Will fix now 👍

Reply:
Resolved in https://github.com/rapid7/metasploit-framework/pull/16382/commits/f7c271aaf4188adf008bb39abfc7d2838051a059